src/tests/components/SUSHIControls.test.js

```diff
@@ -289,3 +290,31 @@ it('shows an error when the FSH file is too long to share', async () => {
     expect(generateLinkSpy).toHaveBeenCalled();
   });
 });
+
+it('opens an example page and renders from config when example button is clicked', async () => {
+  const onClick = jest.fn();
+  const resetLogMessages = jest.fn();
+
+  const { getByText } = render(
+    <HashRouter>
+      <SUSHIControls onClick={onClick} text={'Edit FSH Here'} resetLogMessages={resetLogMessages} />, container
+    </HashRouter>
+  );
+  act(() => {
+    const exampleBtn = getByText('Examples');
+    fireEvent.click(exampleBtn);
+  });
+  const textElement = getByText(/Use our pre-created examples to learn FSH and get swimming!/i);
+  const textElement1 = getByText(/Group 1/i);
+  const textElement2 = getByText(/Group 2/i);
+  const textElement3 = getByText(/Hello World/i);
+  const textElement4 = getByText(/Start Swimming/i);
+  const textElement5 = getByText(/Veternarian/i);
```

Comment:
I was thinking about this again and this test will fail as soon as we fill in real examples. I'm thinking it might be worth it to pass in `config` as a prop to `SUSHIConfig` so that we can mock out the config in the test here. Then we'll have full control over the test and things won't break when we change the examples.

The other idea would be if we can mock out that import here, and then we don't have to change the component props at all.

We can also talk about if this is one we want to do now or later!